video_privacy_enhancer/video_privacy_enhancer.py

```diff
@@ -115,38 +117,30 @@ def download_thumbnail(video_id_from_shortcode, video_thumbnail_url, video_servi
+		six.moves.urllib.request.urlretrieve(video_thumbnail_url, pelican_output_path + "/" + output_directory_for_thumbnails + "/" + video_service_name + "_" + video_id_from_shortcode + ".jpg") # Download the thumbnail. This follows the instructions at http://www.reelseo.com/youtube-thumbnail-image/ for downloading YouTube thumbnail images.
 
 
 # A function to read through each page and post as it comes through from Pelican, find all instances of a shortcode (e.g., `!youtube(...)`) and change it into an HTML <img> element with the video thumbnail.
 # 'dictionary_of_services' below should be the dictionary defined in the settings above, which includes the service's name as the dictionary key, and, for each service, has a dictionary containing 'shortcode_to_search_for_not_including_exclamation_point' and 'function_for_generating_thumbnail_url'
 def process_shortcodes(data_passed_from_pelican):
 	dictionary_of_services = supported_video_services # This should be defined in the settings section above.
 	
-	# Good for debugging:
-	logger.debug("Video Privacy Enhancer Plugin: Using the following dictionary of video services:")
-	logger.debug(dictionary_of_services)
-	
 	if not data_passed_from_pelican._content: # If the item passed from Pelican has a "content" attribute (i.e., if it's not an image file or something else like that). NOTE: data_passed_from_pelican.content (without an underscore in front of 'content') seems to be read-only, whereas data_passed_from_pelican._content is able to be overwritten. This is somewhat explained in an IRC log from 2013-02-03 from user alexis to user webdesignhero_ at https://botbot.me/freenode/pelican/2013-02-01/?tz=America/Los_Angeles.
 		return # Exit the function, essentially passing over the (non-text) file.
 	
 	# Loop through services (e.g., youtube, vimeo), processing the output for each:
-	for video_service_name, video_service_information in dictionary_of_services.iteritems():
+	for video_service_name, video_service_information in six.iteritems(dictionary_of_services):
 		
 		# Good for debugging:
-		logger.debug("Video Privacy Enhancer Plugin: Currently processing the following video service information:")
-		logger.debug(video_service_name)
-		
 		logger.debug("Video Privacy Enhancer Plugin: The name of the current service being processed is '" + video_service_name + "'")
 		
 		shortcode_to_search_for_not_including_exclamation_point = video_service_information['shortcode_not_including_exclamation_point']
 		logger.debug("Video Privacy Enhancer Plugin: Currently looking for the shortcode '" + shortcode_to_search_for_not_including_exclamation_point + "'")
 		
 		function_for_generating_thumbnail_url = video_service_information['function_for_generating_thumbnail_url']
-		logger.debug("Video Privacy Enhancer Plugin: Using the following function name to get thumbnails for the current video service:")
-		logger.debug(function_for_generating_thumbnail_url)
```

Comment:
I realized that it was throwing an error in the debug output. When I originally wrote it a couple of years ago, I thought that it would output the name of the function being used; as I was putting this PR together, though, I realized that it doesn't. Thus, I removed it to clean things up while I was thinking about it.